### ISSUE TITLE
Light controller Showtec Lampy

### ISSUE THREAD

**Issue by JackOlanders** [bitfocus/companion-module-requests#1350]
Light controller Showtec Lampy

**Comment by denishessberger** [comment]
I'm working on it, but it may still take a while.

**Comment by kevin-256** [comment]
I've released version 1.0.0. You can install it manually now, or wait until it's included in Companion.

https://github.com/kevin-256/companion-module-showtec-lampy

**Comment by JackOlanders** [comment]
Thank you so much.
I'll try to test it asap.

Il dom 12 lug 2026, 23:49 Kevin ***@***.***> ha scritto:

**Comment by peternewman** [comment]
> I've released version 1.0.0. You can install it manually now, or wait until it's included in Companion.
> 
> https://github.com/kevin-256/companion-module-showtec-lampy

I assume you're already aware @kevin-256 but see https://companion.free/for-developers/module-development/module-lifecycle/releasing-your-module for how to get your pre-existing module merged into Companion!

**Comment by kevin-256** [comment]
Yeah, i've already submitted it, i'm waiting for approval from the companion module's team
